schema.sql

```diff
@@ -7,9 +7,52 @@ CREATE TABLE animals (
     escape_attempts integer,
     neutered boolean,
     weight_kg decimal,
-    species varchar(255)
+    species_id integer
 );
 
 -- Alter to add new column
 ALTER TABLE animals
-ADD COLUMN species varchar(255);
\ No newline at end of file
+ADD COLUMN species varchar(255);
+
+-- owners table
+CREATE TABLE owners (
+    id serial PRIMARY KEY,
+    full_name varchar(255),
+    age integer
+);
+
+-- species table
+CREATE TABLE species (
+    id serial PRIMARY KEY,
+    name varchar(255)
+);
+
+-- Removing species column
+ALTER TABLE animals
+DROP COLUMN species;
+
+-- Adding species_id column
+ALTER TABLE animals
+ADD COLUMN species_id varchar(255)
```

Comment:
- [ ] Please replace `species_id varchar(255)` with `species_id Integer` because ids are always integers so kindly fix this issue :slightly_smiling_face: